packages/client/hmi-client/src/workflow/ops/simulate-ensemble-ciemss/tera-simulate-ensemble-node-ciemss.vue

```diff
@@ -57,13 +52,10 @@ const props = defineProps<{
 }>();
 const emit = defineEmits(['append-output', 'update-state', 'open-drilldown']);
 
-const showSpinner = ref(false);
-const completedRunId = ref<string>();
-const runResults = ref<RunResults>({});
-const simulationIds: ComputedRef<any | undefined> = computed(
-	<any | undefined>(() => props.node.outputs[0]?.value)
-);
-const progress = ref({ status: ProgressState.Retrieving, value: 0 });
+// const runResults = ref<RunResults>({});
+const runResults = ref<{ [runId: string]: RunResults }>({});
+const inProgressSimulationId = computed(() => props.node.state.inProgressSimulationId);
+const selectedRunId = ref<any>(null);
```

Comment:
Any reason this isnt a ref<string> ? 
Not an issue more just curious

Reply:
fixed